src/pages/components/pagination.js

```diff
@@ -10,12 +17,19 @@ export default function Index({ limit, offset, setLimit, setOffset, data }) {
     offset = data ? (data.length < limit ? offset : offset + limit) : 0;
     setOffset(offset);
   };
+
+  const LimitOptions = () => {
+    return perPage.map((number, Index) => (
+      <Dropdown.Item key={Index} onClick={() => setLimit(number)}>
```

Comment:
1. Do not use inline functions. They are not good for performance
2. Do not use index as key